CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ and this project adheres to
   [#4741](https://github.com/firecracker-microvm/firecracker/pull/4741),
   [#4746](https://github.com/firecracker-microvm/firecracker/pull/4746): Added
   official support for 6.1 microVM guest kernels.
+- [#4763](https://github.com/firecracker-microvm/firecracker/pull/4763): Migrated
```

Comment:
Could you run `tools/devtool checkstyle`, I think `mdformat` wants to format this differently. 

Also, our development practices require some description of the commit in the git commit body. See https://buildkite.com/firecracker/firecracker-pr/builds/10907#0191b1ca-d4ec-47be-965a-f79d5d483664/62-182

Reply:
My `build` directory got corrupted somehow and made mdformat crash, after which I had to delete it entirely and then it ran.

Description of the commit I don't understand though. At the end of the day, all of this can just be squashed before merge with a detailed description, or am I wrong?